config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,42 @@
+require 'carrierwave/orm/activerecord'
+
+def setup_for_storing_assets_in_file_system
+  CarrierWave.configure do |config|
+    config.permissions           = 0600
+    config.directory_permissions = 0700
+    config.storage               = :file
+    config.asset_host            = Settings.url_with_port
+  end
+end
+
+def setup_for_storing_assets_in_s3
+  CarrierWave.configure do |config|
+    config.root            = Rails.root.join('tmp')
+    config.fog_credentials = {
+        provider:              'AWS', # required
+        aws_access_key_id:     Settings.aws_access_key_id, # required
+        aws_secret_access_key: Settings.aws_secret_access_key, # required
+        region:                'us-west-2' # required
```

Comment:
Remove the comments like `required`. That's noise and does not add any meaningful value to code.